### ISSUE TITLE
License for governance documents

### ISSUE THREAD

**Issue by clarfonthey** [florence-social/governance#1]
We need to decide upon a license for our documents, so people can reuse and remix them without having to ask for permission.

GNU documentation licenses, Creative Commons Attribution, and Creative Commons Zero (public domain) are the main options, but there might be something else too.

**Comment by 1011X** [comment]
For licenses like CC-BY, the [Creative Commons license wiki page](https://en.wikipedia.org/w/index.php?title=Creative_Commons_license&oldid=888222259#Attribution) says that attribution must be given to "the best of [one's] ability using the information available", and gives a list of what a user would need to do to respect the license. Almost all of those I think we can do ourselves for convenience.

The [GNU Free Documentation License](https://en.wikipedia.org/w/index.php?title=GNU_Free_Documentation_License&oldid=881464981#Conditions) seems to have quite a few conditions. Some aren't a big deal, but others can get very specific or restrictive about what a user must do (like having to use the same license), and *might* become a burden later on.

In contrast, CC0 has no requirements. Neither party would have to do anything about anything, so it would create the least friction in adoption.

There are also [other CC license requirements we can include](https://en.wikipedia.org/w/index.php?title=Creative_Commons_license&oldid=888222259#Types_of_licenses) based on what we want. For example:
+ Should adopters be able to use these documents for commercial purposes?
+ Should adopters be able to remix or create derivative works?
  + If so, should adopters be able to license derivatives under a more restrictive license?

In my opinion, if these documents catch on, having at least an attribution requirement will really help in raising awareness of the project. I also think commercial use and derivative works should be allowed, so that similar projects can maintain themselves monetarily and change the documents to fit their use cases. I don't have any opinions about the last question, so I'd love to see what someone else thinks.

**Comment by jhaye** [comment]
I agree that keeping things non-restrictive for governance documents is a good idea.

* [CC BY-ND](https://creativecommons.org/licenses/by-nd/4.0/) is too restrictive for collaboratively working and disallows any kind of adaptation
* [CC BY-SA](https://creativecommons.org/licenses/by-sa/4.0/) and [CC BY-NC-SA](https://creativecommons.org/licenses/by-nc-sa/4.0/) are mutually exclusive, which can cause a lot of headaches

A simple non-commercial clause usually hurts more than it helps, for example one wouldn't be allowed to print licensed documents and then charge money to make up for the costs.

I agree with the assessment that CC-0 or CC-BY are the best options here. I'd favour CC-BY for the reason given by @1011X.

Allowing more restrictive licensing for something like governance documents is totally fine as far as I'm concerned.
If there are people in a similar situation as we are, they wouldn't really have an incentive to license more restrictively. The only reason for doing it would be if remixing with another source that requires more restrictive licensing is desired. In that case disallowing that would be of little gain to us.
